### PR TITLE
Fixes for deltalake

### PR DESCRIPTION
**Pull request by PatrickRudolph**

### CODE REVIEW

File: pkg/test/memory.go [hunk 1]
```diff
@@ -335,7 +343,7 @@ func TXTReservedInE820(txtAPI hwapi.APIInterfaces, config *tools.Configuration)
 
 // TXTTPMDecodeSpaceIn820 checks if TPMDecode area is marked as reserved in e820 map
 func TXTTPMDecodeSpaceIn820(txtAPI hwapi.APIInterfaces, config *tools.Configuration) (bool, error, error) {
-	res, err := txtAPI.IsReservedInE820(uint64(tools.TxtTPMDecode), uint64(tools.TxtTPMDecode+0x10000))
+	res, err := txtAPI.IsReservedInE820(uint64(tools.TxtTPMDecode), uint64(tools.TxtTPMDecode+0x5000-1))
```
Comment (ChriMarMe) [inline pkg/test/memory.go:346]: uint64(tools.TxtTPMDecode+0x5000-1))

--------------------------------------------^
This offset looks like a magic number to me.

**Review by ChriMarMe — APPROVED** [review]

File: pkg/test/memory.go (hunk 1 above)
Comment (PatrickRudolph) [reply pkg/test/memory.go]: done

File: pkg/test/acpi.go [hunk 2]
```diff
@@ -487,30 +489,267 @@ type ACPIMADT struct {
 	ACPIHeader
 	LapicAddress uint32
 	Flags        uint32
-	// TODO: variable interrupt controller structures
+	// ariable interrupt controller structures
```
Comment (ChriMarMe) [inline pkg/test/acpi.go:492]: variable

**Review by ChriMarMe — APPROVED** [review]

File: pkg/test/acpi.go (hunk 2 above)
Comment (PatrickRudolph) [reply pkg/test/acpi.go]: Done

**Pull request merged**